lang/languages.xml

```diff
@@ -43,4 +43,5 @@ List of available languages for your wallet's seed:
         <language display_name="Slovensky" locale="sk_SK" wallet_language="English" flag="/lang/flags/slovakia.png" qs="none"/>
         <language display_name="Arabic" locale="ar_AR" wallet_language="English" flag="/lang/flags/egypt.png" qs="none"/>
         <language display_name="Slovenski" locale="sl_SI" wallet_language="English" flag="/lang/flags/slovenia.png" qs="none"/>
+		<language display_name="Português (PT)" locale="pt-pt_PT" wallet_language="Português (PT)" flag="/lang/flags/portugal.png" qs="none"/>
```

Comment:
on wallet_language use "Português" and align the strings with the above ones

Reply:
In my notepad++ looks ok, will see what went wrong.